lib/main.coffee

```diff
@@ -49,8 +38,8 @@ module.exports =
         filePath = textEditor.getPath()
         command = @executablePath
         ruleset = @rulesets
-        if @projectRules
-          rulesetPath = helpers.find(filePath, 'ruleset.xml')
+        if /^[a-z0-9]+\.xml$/gi.test(@rulesets)
```

Comment:
You're welcome :)
Yes indeed it's breaking. I tried to find examples on how to create a migration but I couldn't find any.
Which is a shame since a migration shouldn't be harder than updating the `rulesets` config to `ruleset.xml` for users that have `projectRules` checked 😞 

It happens 😂